salt/modules/zypper.py

```diff
@@ -21,6 +21,10 @@
 import os
 import time
 import datetime
+try:
+    from salt.utils.versions import LooseVersion
+except ImportError:
+    LooseVersion = None
```

Comment:
Well, to be honest, I don't think that we'll ever hit the ``TypeError`` since we should never fail to import. So it's probably harmless, but unnecessary. We don't guard this sort of import elsewhere in the codebase, so I thiink you're save in removing the try/except here.